racket/collects/ffi/unsafe.rkt

```diff
@@ -1410,10 +1410,11 @@
 ;; ----------------------------------------------------------------------------
 ;; Struct wrappers
 
-(define (compute-offsets types alignment declared)
+(define* (compute-offsets types [alignment #f] [declared '()])
```

Comment:
I'm not seeing a `provide` of `compute-offsets`. Am I overlooking something?

Reply:
The module (for some reason) defined the `define*` macro to be a define and provide. The other cstruct parts of that module used it for define, which is why I simply added the `*`.

Reply:
Ah, right.